src/shared/components/instances.tsx

```diff
@@ -27,22 +30,35 @@ export class Instances extends Component<any, any> {
 
     var recommended = [];
     var remaining = [];
+    var values = [];
+
     for (var i of instance_stats.stats.instance_details) {
       if (recommended_instances.indexOf(i.domain) > -1) {
         recommended.push(i);
       } else {
         remaining.push(i);
       }
+
+      // Min number of active users is 10 to include in this calculation
+      if (i.site_info.site_view.counts.users_active_month > 10) {
+        values.push(i.site_info.site_view.counts.users_active_month);
+      }
     }
-    // shuffle recommended instances list into random order
-    // https://stackoverflow.com/a/46545530
+
+    // Use these values for the shuffle
+    const max = Math.max(...values);
+    const min = Math.min(...values);
+    const avg = this.averageFunc(values);
```

Comment:
Ah so this is calculating `averageMonthlyUsers`, `maxMonthlyUsers` and `minMonthlyUsers` across all instances. Can you rename the vars to clarify this?

Reply:
Renamed.